web/config/builder.go

```diff
@@ -104,9 +106,23 @@ func (b *WebBuilder) Init(v *viper.Viper, outbox chan<- mail.Mail) *WebBuilder {
 	b.Outbox = outbox
 	b.GrantScopes = b.getGrantScopesFromFile(flags.ScopesFilePath)
 	b.HydraHelper = new(hydra.DefaultHydraHelper).Init(b.HydraAdminURL)
-	b.Self = new(client.WhisperClient).InitFromParams(flags.HydraAdminURL, flags.HydraPublicURL, "whisper", "", b.GrantScopes.GetScopeListFromGrantScopeMap(), []string{})
 	b.DB = b.initDB()
 
+	hydraAdminURI, err := url.Parse(flags.HydraAdminURL)
+	gohtypes.PanicIfError("Invalid hydra admin url", 500, err)
+	hydraPublicURI, err := url.Parse(flags.HydraPublicURL)
+	gohtypes.PanicIfError("Invalid hydra public url", 500, err)
+
+	b.Self = new(client.WhisperClient).InitFromConfig(&config.Config{
+		ClientID:       "whisper",
+		ClientSecret:   "",
+		WhisperURL:     nil,
+		HydraAdminURL:  hydraAdminURI,
+		HydraPublicURL: hydraPublicURI,
+		Scopes:         b.GrantScopes.GetScopeListFromGrantScopeMap(),
+		RedirectURIs:   []string{},
+	})
+
```

Comment:
@claudiosegala, this update should be done in another PR. Please rollback the `whisper-client` lib version to the previous one.